### PR TITLE
Feature/saml e2e tests

### PR DESCRIPTION
**Pull request by sebastianmichalski**
### Description
Added SAML e2e tests and github actions environment setup for those tests

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Log in via OIDC', () => {
+  const login = 'admin';
```
Comment (kavilla) [inline cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js:7]: https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress.json#L16

I think you can pull from the cypress env variable for these as well unless you want to have a separate cypress variable defined as like `OIDC_USERNAME`

**Comment by kavilla** [comment]
Hello, @sebastianmichalski please respond to the above and then re-open.

File: cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js (hunk 1 above)
Comment (sebastianmichalski) [reply cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js:7]: @jakubp-eliatra can You take care of frontend part?

File: cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    cy.get('button[data-test-subj^="log-out-"]').click();
+    cy.get('#kc-page-title').should('be.visible');
+  };
+
+  afterEach(async () => {
+    logout();
+  });
+
+  it('Login to app/opensearch_dashboards_overview#/ when OIDC is enabled', () => {
+    cy.visit('http://localhost:5601/app/opensearch_dashboards_overview#/');
```
[the remark this answers is not in the document]
Comment (sebastianmichalski) [reply cypress/integration/plugins/security-dashboards-plugin/auth_test.cy.js:29]: @jakubp-eliatra can You take care of frontend part?

**Pull request closed without merging**